src/test/jtx/TestHelpers.h

```diff
@@ -27,17 +27,25 @@
 #include <ripple/protocol/jss.h>
 #include <test/jtx/Env.h>
 
-#include <ranges>
+#include <vector>
 
 namespace ripple {
 namespace test {
 namespace jtx {
 
 // Helper to make vector from iterable
+template <typename T>
+concept iterable = requires(T& v)
+{
+    std::begin(v);
+    std::end(v);
+};
+
+template <typename Input>
 auto
-make_vector(auto const& input) requires std::ranges::range<decltype(input)>
+make_vector(Input const& input) requires iterable<Input>
```

Comment:
I'm not able to find usages of this helper function in this PR, have you used it in another PR?

Reply:
it is used in `PerfLog_test.cpp` and `Handler_test.cpp`

Reply:
okay, I didn't find these results a few days ago, thanks 👍